src/slashcommands/Birthday.ts

```diff
@@ -117,6 +133,14 @@ export class Birthday implements SlashCommand {
 			});
 		}
 
+
+		if (interaction.options.getInteger('day')! > dayCap[interaction.options.getString('month')!] || interaction.options.getInteger('day')! < 0){
```

Comment:
Currently this line is comparing the entered date < 0.  This will allow users to set their birthday to the 0th of each month.

Please fix accordingly

Reply:
done